pkg/logs/input/file/file_provider.go

```diff
@@ -63,9 +64,10 @@ func (p *Provider) FilesToTail(sources []*config.LogSource) []*File {
 		source := sources[i]
 		tailedFileCounter := 0
 		files, err := p.CollectFiles(source)
+		wildcardPath := p.containsWildcard(source.Config.Path)
```

Comment:
nits: its unclear that the type of this var is `bool`

Reply:
what I am suggest is `isWildcard` or something like that

Reply:
Updated